File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        for ch in allowed:
            for i in range(len(words)):
                 if ch in words[i]:
                    words[i] = words[i].replace(ch,"")
        count=0
        for word in words:
            if len(word)== 0:
                count+=1
        return count

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        set<int> all(allowed.begin(),allowed.end());
        int count =0;
        for(string word:words){
            for(char ch:word){
                if(!all.contains(ch)){
                    count++;
                    break;
                }
            }
        }
        return words.size() - count;
    }
};